assembly/fabric8-stacks/src/main/resources/stacks.json

```diff
@@ -141,7 +141,7 @@
   {
     "id": "che7-preview-plugin-dev",
```

Comment:
Why haven't you updated the id here? Is it because it would duplicate the stack in prod?

Reply:
I was able to see this one into prod so I kept the id
you think that I need to rename id here ?

Reply:
I would change it just to be consistent with the dev stack. And maybe to be sure that it gets updates. But honestly I am not familiar with the stack update mechanism.